test/lit/passes/optimize-instructions-atomics.wast

```diff
@@ -0,0 +1,33 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py
```

Comment:
Does this mean they should not be updated by hand?  Perhaps mention that explictly?